app/helper_functions.py

```diff
@@ -0,0 +1,63 @@
+from app import db
+from sqlalchemy import desc, asc
+from flask import make_response, abort, request
+from app.models.task import Task
+from app.models.goal import Goal
+import requests
+import os
+
+## Helper Functions
+# Validate ID function
+def validate_model(cls, model_id):
+    model_item = cls.query.get(model_id)
+
+    if not model_item:
+        return abort(make_response({"message": f"{cls.__name__} with ID {model_id} not found."}, 404))
+
+    return model_item
+
+# Sort results
+def query_sort(cls):
+    query_sort = request.args.get("sort")
+
+    if query_sort == "asc":
+        model_items = db.session.query(cls).order_by(asc(cls.title)).all()
+    elif query_sort == "desc":
+        model_items = db.session.query(cls).order_by(desc(cls.title)).all()
+    else:
+        model_items = cls.query.all()
```

Comment:
Nice work on getting the database to do the heavy lifting when it came to sorting the tasks!